lib/SiftResponse.php

```diff
@@ -13,6 +13,11 @@ public function __construct($result, $httpStatusCode, $request) {
         $this->apiStatus = null;
         $this->apiErrorMessage = null;
 
+        // Check for errors and bad httpStatusCode
```

Comment:
I'd rather propagate this into `SiftResponse` by adding a new, optional parameter to `__construct()`.